src/Crocodealer/Command/Sync.hs

```diff
@@ -0,0 +1,81 @@
+module Crocodealer.Command.Sync
+       ( sync
+       ) where
+
+import Network.HTTP.Simple (Response, getResponseBody, getResponseStatus, httpBS, parseRequest)
+import Network.HTTP.Types.Status (Status (..))
+
+import Crocodealer.Command.Colors (errorMessage, infoMessage, warningMessage)
+
+data File = File
+    { fileName     :: !Text
+    , fileContents :: !Text
+    }
+
+data FileStatus
+    = UpToDate
+    | Outdated
+    | FileDoesNotExist
+
+data Repo = Repo
+    { repoUser   :: !Text
+    , repoName   :: !Text
+    , repoStatus :: !FileStatus
+    }
+
+mkRepo :: Text -> Text -> FileStatus -> Repo
+mkRepo user name status =
+    Repo { repoUser = user, repoName = name, repoStatus = status }
+
+sync :: Text -> Text -> Text -> [Text] -> IO ()
+sync user repoName fileName repositories = do
```

Comment:
It's better to have newtypes instead of many arguments of the same type. Let's create module `Crocodealer.Core.GitHub` which will have `UserName`, `RepoName` newtypes, and use them here. Just to have more type-safe interface.

Reply:
Also, `repoName` is a bit confusing name. Let's rename variable to `templateRepoName`.